src/ASM/ASMs2Dmx.C

```diff
@@ -1205,3 +1205,13 @@ void ASMs2Dmx::getBoundaryNodes (int lIndex, IntVec& nodes, int basis,
     for (size_t b = 1; b <= this->getNoBasis(); ++b)
       this->ASMs2D::getBoundaryNodes(lIndex, nodes, b, thick, 0, local);
 }
+
+
+bool ASMs2Dmx::updateCoords (const Vector& displ)
+{
+  if (!surf) return true; // silently ignore empty patches
+  if (shareFE) return true;
+
+  surf->deform(displ,nsd);
+  return true;
+}
```

Comment:
Suggest that the sanity check in ASMs2D is changed from `!=` to `<` and you don't have to do this reimplementation.
Similar for 1D and 3D.

Reply:
My main point was that the ASMs2Dmx overload was not needed when changing the sanity check in ASMs2D, but it appears to still be in this commit?

Reply:
remedied.